src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -93,6 +113,7 @@ const CooperationDetails = () => {
       <Box>
         <StatusChip status={response.status} />
         <TitleWithDescription
+          key={Date.now()}
```

Comment:
using Date as a key is a bad practice. Generate some id for this case

Reply:
fixed